server/routes/serviceProviderReferrals/serviceProviderReferralsController.test.ts

```diff
@@ -107,10 +107,10 @@ describe('GET /service-provider/referrals/:id', () => {
     communityApiService.getServiceUserByCRN.mockResolvedValue(serviceUser)
 
     await request(app)
-      .get(`/service-provider/referrals/${sentReferral.id}`)
+      .get(`/service-provider/referrals/${sentReferral.id}/details`)
       .expect(200)
       .expect(res => {
-        expect(res.text).toContain('Who do you want to assign this accommodation referral to?')
```

Comment:
Why have we changed this text? Would be good to see it mentioned in a commit message if intentional.